src/main/java/org/isf/dicom/gui/DicomGui.java

```diff
@@ -205,7 +205,8 @@ private void initComponents() {
 		jButtonDeleteDicom.setName("jButtonDeleteDicom");
 		jButtonDeleteDicom.setEnabled(false);
 
-		jButtonExit.setText(MessageBundle.getMessage("angal.common.close"));
+		jButtonExit.setText(MessageBundle.getMessage("angal.common.close.btn"));
+		jButtonExit.setMnemonic(MessageBundle.getMnemonic("angal.common.close.btn.key"));
```

Comment:
Exit or Close is the same, maybe needs refactoring like others

Reply:
Thanks; done.